src/View/Cell.php

```diff
@@ -259,15 +260,17 @@ protected function _cacheConfig($action)
      * This is because PHP will not allow a __toString() method to throw an exception.
      *
      * @return string Rendered cell
+     * @throws \Error Include error details for PHP 7 fatal errors.
      */
     public function __toString()
     {
         try {
             return $this->render();
         } catch (Exception $e) {
             trigger_error(sprintf('Could not render cell - %s [%s, line %d]', $e->getMessage(), $e->getFile(), $e->getLine()), E_USER_WARNING);
-
             return '';
+        } catch (Error $e) {
+            throw new Error(sprintf('Could not render cell - %s [%s, line %d]', $e->getMessage(), $e->getFile(), $e->getLine()));
```

Comment:
@thinkingmedia First of all, now I think your change is not a bad idea. So I don't hold to my idea. It is an idea to keep the stacktrace and to show the same message as PHP5.

Regarding to ToStringTrait, if there is no try/catch block in __toString(), you would get unhelpful messages. For example:
```php
echo new class {
    public function __toString() {
        // Autload X.php that contains syntax error.
        return X::someting();
    }
};
```
> Fatal error: Method A::__toString() must not throw an exception, caught ParseError: syntax error, unexpected end of file, expecting identifier (T_STRING) in /path/to/main.php on line 0

You will see a wrong file path and line number zero.

Regarding to catch(Exception $e) block in my example, I removed trigger_error intentionally, as I don't like the current behavior. It is not good to suppress important exceptions such as PDOException. In such cases, applications should send HTTP 500 error to the client. Otherwise the unexpected response will be cached by search engines or proxy cache servers that the client are using.

Although I have never used Cell, I would need to use it in the feature, as requestAction is deprecated. It was a very easy way to display a HTML fragment being changed via AJAX, on the first render.

But as I mentioned above, I don't think this is an issue to specific Cells. It is an annoying issue in PHP7. Seeing bugs.php.net, this issue might be fixed in the near future. Related tickets are updated recently.
https://bugs.php.net/bug.php?id=48622
https://bugs.php.net/bug.php?id=53648